src/Controls/src/Core/Shapes/Rectangle.cs

```diff
@@ -42,8 +42,11 @@ protected override void OnPropertyChanged([CallerMemberName] string propertyName
 			base.OnPropertyChanged(propertyName);
 
 			if (propertyName == RadiusXProperty.PropertyName ||
```

Comment:
Only happens in `Rectangle` and `RoundRectangle`? If happens with all the Shapes can apply changes in the ShapeViewHandler to invoke the `InvalidateShape` method changing the visibility.